ios/FluentUI/Core/Theme/Tokens/ShadowInfo.swift

```diff
@@ -59,4 +60,63 @@ public struct ShadowInfo: Equatable {
 
     /// The vertical offset of the shadow for shadow 2.
     public let yTwo: CGFloat
+
+    /// The number that the figma blur needs to be divided by to properly display shadows
+    private let blurDivisor: CGFloat = 2.0
```

Comment:
Optional - in FURN we called this appleShadowBlurAdjustment and did * 0.5 instead of /2.0. Up to you if we want to be more aligned or not, the 'apple' part doesn't make as much sense here but maybe it could be just 'shadowBlurAdjustment'